template.env

```diff
@@ -1,7 +1,7 @@
 # Random dummy private keys by default
-POLYGON_PRIVATE_KEY_MUMBAI=806afb0854acc07ffd9d0adf796c864981ab91b5e24e256824e3e8b0f1b7edb7
-POLYGON_PRIVATE_KEY_MAINNET=806afb0854acc07ffd9d0adf796c864981ab91b5e24e256824e3e8b0f1b7edb7
-ETHEREUM_PRIVATE_KEY_RINKEBY=806afb0854acc07ffd9d0adf796c864981ab91b5e24e256824e3e8b0f1b7edb7
-ETHEREUM_PRIVATE_KEY_MAINNET=806afb0854acc07ffd9d0adf796c864981ab91b5e24e256824e3e8b0f1b7edb7
+POLYGON_PRIVATE_KEY_MUMBAI=0adc709aaa843e43ec554ea01e6deb1223701593b42043cf88932c7b016583a7
```

Comment:
Why change this? :D

Reply:
private key leaked!!

Reply:
Haha.. its localhost key for testing initially. Will change it back with previous one.

Reply:
Solved